packages/ds-medicare-gov/src/styles/components/_override.help_drawer.scss

```diff
@@ -2,31 +2,15 @@
 $mct-drawer-close-button-size: 26px;
 $mct-drawer-heading-font-size: $h1-mobile-font-size;
 
-.mct-c-help-drawer {
-  .ds-c-drawer__header {
-    background-color: $drawer-header__background-color;
-  }
-
-  .ds-c-drawer__header-heading {
-    font-family: $font-family-montserrat;
-    font-size: $mct-drawer-heading-font-size;
```

Comment:
I used the xl font size defined in our tokens. It's not an exact match (I think it's off by 4px), but it's the closest they have without going too far over size. Mgov offers: `$font-size-xl: 24px` and `$font-size-2xl: 32px` sizes (I chose 24px); if they need a 28px equivalent, that decision should probably happen on the token level imo.

Reply:
Just dawned on me today, Carmen's filing a lot of tickets to get the mgov theme to align with MAX and it _seems_ MAX is using a different type scale than the DS. Is that something we need to straighten out?